modules/cluster-autoscaler-cr.adoc

```diff
@@ -48,16 +48,22 @@ you assign to each pod.
 <4> Specify the maximum number of cores to deploy.
 <5> Specify the minimum amount of memory, in GiB, per node.
 <6> Specify the maximum amount of memory, in GiB, per node.
-<7> Specify the type of GPU node to deploy. Only `nvidia.com/gpu` and `amd.com/gpu`
-are valid types.
+<7> Optionally, specify the type of GPU node to deploy. Only
+`nvidia.com/gpu` and `amd.com/gpu` are valid types.
 <8> Specify the minimum number of GPUs to deploy.
 <9> Specify the maximum number of GPUs to deploy.
-<10> Specify whether the ClusterAutoscaler can remove unnecessary nodes.
-<11> Specify the period, in minutes, to wait before deleting a node after
-a node has recently been _added_.
-<12> Specify the period, in minutes, to wait before deleting a node after
-a node has recently been _deleted_.
-<13> Specify the period, in seconds, to wait before deleting a node after
-a scale down failure occured.
-<14> Specify the period, in seconds, before an unnecessary node is eligible
-for deletion.
+<10> In this section, you can specify the periods in by using any valid
```

Comment:
"... specify the periods by using ..."

Reply:
Though given the reference is to pkg/time/ParseDuration I wonder whether we should s/period/duration.

Reply:
They're synonyms, so normally I'd agree, but rewriting the rest of the statements to use "duration" instead of "period" make them a bit clunky. I think that "the period to wait for each action by using" might be most consistent with the subsequent content.